modules/ROOT/pages/java-embedded/setup.adoc

```diff
@@ -83,7 +83,7 @@ The examples included here use the top-level artifact approach.
 ====
 The examples below are only valid Neo4j Community Edition.
 
-To add Neo4j Enterprise Edition as a dependency, please get in contact with Neo4j Professional Services.
+To add Neo4j Enterprise Edition as a dependency, please get in contact with link:https://neo4j.com/contact-us/[Neo4j Professional Services^].
```

Comment:
You don't need the character ^. You can also simplify it further if you want by replacing "please get in contact with" with  just "contact", like:
```suggestion
To add Neo4j Enterprise Edition as a dependency, contact link:https://neo4j.com/contact-us/[Neo4j Professional Services].
```